cabby/geo/util.py

```diff
@@ -331,6 +382,77 @@ def check_if_geometry_in_polygon(geometry: Any, poly: Polygon) -> Polygon:
     geometry['geometry'].intersects(poly)
 
 
+def point_from_str_coord(coord_str: Text) -> Point:
+  '''Converts coordinates in string format (latitude and longtitude) to Point. 
+  E.g, of string '(40.715865, -74.037258)'.
+  Arguments:
+    coord: A lat-lng coordinate to be converted to a point.
+  Returns:
+    A point.
+  '''
+  list_coords_str = coord_str.replace("(", "").replace(")", "").split(',')
+  coord = list(map(float, list_coords_str))
+
+  return Point(coord[1], coord[0])
+
+
+def coords_from_str_coord(coord_str: Text) -> Tuple[float, float]:
+  '''Converts coordinates in string format (latitude and longtitude) to 
+  coordinates in a tuple format. 
+  E.g, of string '(40.715865, -74.037258)'.
+  Arguments:
+    coord: A lat-lng coordinate to be converted to a tuple.
+  Returns:
+    A tuple of lat-lng.
+  '''
+  list_coords_str = coord_str.replace("(", "").replace(")", "").split(',')
+  coord = list(map(float, list_coords_str))
+
+  return (coord[0], coord[1])
```

Comment:
I know that this is tricky - but some libraries use latitude-longitude versus some specify longitude,latitude. Is there a way to unify this? e.g L396 versus L411 are confusing unless we see that one uses shapely.Point
Consider using a named tuple